jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/NewActionViewImpl.html

```diff
@@ -38,6 +38,23 @@ <h4 class="modal-title" data-i18n-key="NewAction"></h4>
                                 <span data-field="assignation-help" class="help-block"></span>
                             </div>
                         </div>
+
+                        <div class="form-group" data-field="action-nodetype-group">
+                            <label class="control-label col-md-3" data-i18n-key="NodeType"></label>
+                            <div class="col-md-9">
+                                <input type="text" data-field="action-nodetype-input" class="form-control"/>
+                                <span data-field="nodetype-help" class="help-block"></span>
+                            </div>
+                        </div>
+
+                        <div class="form-group" data-field="action-case-definitions-group">
+                            <label class="control-label col-md-3" data-i18n-key="ProcessId"></label>
```

Comment:
Please add a required indicator to this field. Also perhaps name it as _Process Name_ instead.